### ISSUE TITLE
Not work with installing via Package Control

### ISSUE THREAD

**Issue by YariKartoshe4ka** [YariKartoshe4ka/sublime-svg-viewer#1]
Crashes after reloading with log
```
reloading plugin SVG Viewer.svg-viewer
Traceback (most recent call last):
  File "/opt/sublime_text/sublime_plugin.py", line 125, in reload_plugin
    m = importlib.import_module(modulename)
  File "./python3.3/importlib/__init__.py", line 90, in import_module
  File "<frozen importlib._bootstrap>", line 1584, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1565, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1532, in _find_and_load_unlocked
  File "/opt/sublime_text/sublime_plugin.py", line 1199, in load_module
    exec(compile(source, source_path, 'exec'), mod.__dict__)
  File "/home/yaroslav/.config/sublime-text-3/Installed Packages/SVG Viewer.sublime-package/svg-viewer.py", line 31, in <module>
  File "/home/yaroslav/.config/sublime-text-3/Installed Packages/SVG Viewer.sublime-package/svg-viewer.py", line 34, in SvgViewerViewSvgCommand
NotADirectoryError: [Errno 20] Not a directory: '/home/yaroslav/.config/sublime-text-3/Installed Packages/SVG Viewer.sublime-package/converters.json'
```

**Comment by YariKartoshe4ka** [comment]
Error, because Sublime installs package not as folder, but as zip (sublime-package) archive

**Comment by YariKartoshe4ka** [comment]
It was a bad idea to use raw `open().read()`, because by default, sublime sets the package as packed (zipped) and you can't just get a file from it, so I changed this to `sublime.load_resource`. It is also not possible to edit converters.json, so I create 2 versions: Packages/SVG Viewer/converter.json (default converters), Packages/User/converters.json (custom converters). Patched in [1.0.1](https://github.com/YariKartoshe4ka/sublime-svg-viewer/releases/tag/1.0.1)

**Comment by YariKartoshe4ka** [comment]
This error still appeared and I decided to just add the file .no-sublime-package
